server.js

```diff
@@ -69,6 +69,10 @@ app.post('/', function(req, res) {
       data.pull_request.number // 23
     );
 
+    reviewers = reviewers.filter(function(reviewer){
```

Comment:
This is actually not going to work as expected. If the author is part of the top 3 reviewers, then you're only going to have two. Instead you should pass down the author as a third argument of guessOwnersForPullRequest and do the filtering there.